src/widgets/menu-select/menu-select.tsx

```diff
@@ -85,7 +86,7 @@ const renderer = ({
     refine,
     items,
     instantSearchInstance,
-  }: MenuRendererOptions & RendererOptions<MenuConnectorParams>,
+  }: MenuRenderState & RendererOptions<MenuConnectorParams>,
```

Comment:
I was expecting to see here something like other renderers:

`Renderer< MenuRenderState, Partial<MenuWidgetParams>>`

What's the reason for not following the same pattern?

Reply:
essentially it's the same, but this required less changes. Since this is purely internal, we can mutualise later without breaking change

Reply:
I'm fine either way.